tinyms/model/vgg.py

```diff
@@ -0,0 +1,222 @@
+# Copyright 2021 Huawei Technologies Co., Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+
+import numpy as np
+
+from tinyms import layers, Tensor
+from tinyms.layers import BatchNorm2d, ReLU, Conv2d, MaxPool2d, \
```

Comment:
If u have already imported `layers` module, it should not be necessary to import all these concrete operators.